test/interactions/keyInteractionTests.ts

```diff
@@ -1,302 +1,327 @@
-        keyInteraction.onKeyPress(aCode, aCallback);
+        keyInteraction.onKeyPress(aKeyCode, aKeyCodeCallback);
         keyInteraction.attachTo(component);
 
         TestMethods.triggerFakeMouseEvent("mouseout", component.background(), -100, -100);
-        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode);
-        assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when not moused over the Component");
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aKeyCode);
+        assert.isFalse(aKeyCodeCallbackCalled, "callback for \"a\" was not called when not moused over the Component");
 
-        keyInteraction.offKeyPress(aCode, aCallback);
+        keyInteraction.offKeyPress(aKeyCode, aKeyCodeCallback);
         svg.remove();
       });
 
-      it("removing keyPress callbacks is possible", () => {
-        keyInteraction.onKeyPress(aCode, aCallback);
+      it("can remove keyPress callbacks", () => {
+        keyInteraction.onKeyPress(aKeyCode, aKeyCodeCallback);
         keyInteraction.attachTo(component);
 
         TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
-        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode);
-        assert.isTrue(aCallbackCalled, "callback for \"a\" was called when \"a\" key was pressed");
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aKeyCode);
+        assert.isTrue(aKeyCodeCallbackCalled, "callback for \"a\" was called when \"a\" key was pressed");
 
-        keyInteraction.offKeyPress(aCode, aCallback);
-        aCallbackCalled = false;
-        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode);
-        assert.isFalse(aCallbackCalled, "callback for \"a\" was disconnected from the interaction");
+        keyInteraction.offKeyPress(aKeyCode, aKeyCodeCallback);
+        aKeyCodeCallbackCalled = false;
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aKeyCode);
+        assert.isFalse(aKeyCodeCallbackCalled, "callback for \"a\" was disconnected from the interaction");
 
-        keyInteraction.onKeyPress(aCode, aCallback);
-        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aCode);
-        assert.isTrue(aCallbackCalled, "callback for \"a\" was properly connected back to the interaction");
+        keyInteraction.onKeyPress(aKeyCode, aKeyCodeCallback);
+        TestMethods.triggerFakeKeyboardEvent("keydown", component.background(), aKeyCode);
+        assert.isTrue(aKeyCodeCallbackCalled, "callback for \"a\" was properly connected back to the interaction");
 
-        keyInteraction.offKeyPress(aCode, aCallback);
+        assert.strictEqual(keyInteraction.offKeyPress(aKeyCode, aKeyCodeCallback), keyInteraction,
+          "unsetting the keyPress callback returns the interaction");
```

Comment:
It seems that this test is essentially divided into two parts...  What is the second part trying to test?

Also it seems like the last line here is testing that the setter is chainable; shouldn't this assert be made higher up?